raiden/transfer/mediated_transfer/initiator.py

```diff
@@ -73,14 +74,14 @@ def handle_block(
     secrethash = initiator_state.transfer.lock.secrethash
     locked_lock = channel_state.our_state.secrethashes_to_lockedlocks.get(secrethash)
 
-    lock_expired = channel.is_lock_expired(
+    lock_expiration_threshold = locked_lock.expiration + DEFAULT_NUMBER_OF_CONFIRMATIONS_BLOCK * 2
```

Comment:
I think it would be more correct to have the constant named as `DEFAULT_NUMBER_OF_BLOCK_CONFIRMATIONS` as the current one reads a bit weird.

Reply:
fine with me